other/longestincreasingsubsequencegreedy/longestincreasingsubsequencegreedy.go

```diff
@@ -0,0 +1,43 @@
+package longestincreasingsubsequencegreedy
```

Comment:
This doesn't need a separate package. There is a `dynamic` package in the repository which already has a longest increasing subsequence algorithm. Just move this algorithm there along with the old implementation.

Reply:
Done.